README.md

```diff
@@ -105,10 +105,14 @@ Don't forget to give the project a star!
    pnpm run dev
    ```
 9. There should be newly opened http://localhost:5173/ window in your browser with react as a FE framework communicating with BE Symfony. The green value is fetched from BE database. You are ready to start local development. Happy coding. Maybe time to give this project a [star](https://github.com/petrzivny/symfony-react-skeleton) ⭐, thank you. 😉 
+W: nechybi tady nejaky popis toho, ze ti do toho vleze CORS a ze to neni uplne produkcni nastaveni apky, kdyz to vypnes nebo tak neco?
 
 Next time you only need to perform points 4. and 8. to start developing. I recommend to set up an <a href="#setup-alias-for-fast-start-of-development">alias</a> for them.
+W: nevidel jsem popis jak pridavat dalsi balicky, evidentne to musis delat taky v tom kontajneru ...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+... tady mi nekde chybi obrazek nebo vysvetleni jak to spolu komunikuje ... 
```

Comment:
Co myslíš? FE a BE? Jako obrázek FE <-> Nginx <-> BE? To by byl dost jednoduchý obrázek. Nebo myslíš komplet scháma i s SQL?